### DISCUSSION TITLE
Add repeater limit.(As a way to store extra data all in one pre-structured JSON field)

### DISCUSSION THREAD

**Discussion by development-vargamarcel** [directus/directus#12433] [Feature Requests]
The repeater is an array of pre-structured objects.
**Can we have a repeater that is one pre-structured object only?** maybe not call it that,but that is my point.

Would be useful for extra,not-so-important data in a row instead of adding too many columns as some of those values will never be present or filtered on.

```
example table,catTable:
catTable.extra_info={
likesPlayingGolf:true,
friends:[A,B,C],
prefferedFood:[G,H,R,E],
legsIfGraterThanFour:7,
OTHER:OTHERVALUE
...
}
```

As you can see some of the info above may not even be present,and surely is not that important (legsIfGraterThanFour,will likely never be present).

I know strange example.

**Comment by benhaynes** [comment]
So, like an inline form, based on non-relational fields, that saves as field JSON?

**Reply by development-vargamarcel** [reply]
yes,precisely.

**Reply by benhaynes** [reply]
I like it.

**Comment by HCl-not-HCi** [comment]
I also came across this missing feature.
I use an existing application that allows to store additional information about users in an `attribs` column as JSON (for example, I would like to store the language in there). Editing/reading the attributes in that application is not really suitable because there is only a plaintext editor for that. So I thought directus might be a nice alternative here.

However, I figured out that editing JSON based on a specific schema is only possible in the repeater interface. Since I only need the attributes once, the repeater interface does not really make sense for me nor is it expected by the application to handle an array instead of an object.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
